.ci/magician/utility/utils.go

```diff
@@ -23,6 +23,7 @@ import (
 	"math"
 	"net/http"
 	"os"
+	"strings"
```

Comment:
Could you move this file's changes to a separate PR for clarity, since it's not related?

Reply:
This is related and needed. The RounderTripper passes a []Byte object and the other passes and generic go object that needs to be serialized.

Reply:
ah, okay!